### PR TITLE
Fix indent in the SPDX identifier

### LINKED ISSUE #1083: inconsistent indent in the SPDX identifier
Some identifier contains tab or spaces. Let's unify those to spaces

eg) 
``` * @file	neuralnet.cpp``` ->  ```* @file  neuralnet.cpp```

### PR DESCRIPTION
**Pull request by JuYeong0413**
Change tab indents to spaces in the SPDX identifier.
Resolves #1083

Signed-off-by: Juyeong Lee <2jy22@naver.com>

### CODE REVIEW

File: Applications/Embedding/Tensorflow/embedding.py [hunk 1]
```diff
@@ -3,12 +3,12 @@
...
-# @brief	This is Simple Embedding Layer Training Example
-# @see		https://github.com/nnstreamer/nntrainer
-# @author	Jijoong Moon <jijoong.moon@samsung.com>
-# @bug		No known bugs except for NYI items
+# @file	  embedding.py
+# @date	  15 March 2021
+# @brief  This is Simple Embedding Layer Training Example
+# @see    https://github.com/nnstreamer/nntrainer
+# @author Jijoong Moon <jijoong.moon@samsung.com>
+# @bug    No known bugs except for NYI items
```
Comment (JuYeong0413) [inline Applications/Embedding/Tensorflow/embedding.py:11]: I wonder why some parts seem weird in GitHub...
Screen Shot 2021-04-27 at 5 12 15 PM

File: Applications/VGG/jni/main.cpp [hunk 2]
```diff
@@ -2,12 +2,12 @@
...
- * @see		https://github.com/nnstreamer/nntrainer
- * @author	Jijoong Moon <jijoong.moon@samsung.com>
- * @bug		No known bugs except for NYI items
- * @brief	This is VGG Example with
+ * @file   main.cpp
+ * @date   05 Oct 2020
+ * @see    https://github.com/nnstreamer/nntrainer
+ * @author Jijoong Moon <jijoong.moon@samsung.com>
+ * @bug	   No known bugs except for NYI items
+ * @brief  This is VGG Example with
```
Comment (JuYeong0413) [inline Applications/VGG/jni/main.cpp:10]: Screen Shot 2021-04-27 at 5 13 10 PM

.... and some more. Looks like the indent is inconsistent in GitHub, but it is okay in IDE.

File: Applications/Embedding/Tensorflow/embedding.py (hunk 1 above)
Comment (zhoonit) [reply Applications/Embedding/Tensorflow/embedding.py:11]: It looks fine on my github :)
image

**Review by zhoonit — APPROVED** [review]
Nice work :)

I've checked ci and there is no particular issue regarding this PR.

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Pull request merged**